### PR TITLE
feat: port top-level `export` command from `influxdb`

### PR DESCRIPTION
**Pull request by danxmoran**
Addresses part of #12 (still need to add the subcommands).

Most of the strange logic here is in the CLI parsing. I've replicated all of the existing behavior except for one case where filters were silently being converted/ignored; that case is now an error.

### CODE REVIEW

File: api/contract/cli.yml [hunk 1]
```diff
@@ -253,3 +255,11 @@ components:
       $ref: "./openapi/src/common/schemas/Cells.yml"
     Cell:
       $ref: "./openapi/src/common/schemas/Cell.yml"
+    TemplateExport:
+      $ref: "./overrides/schemas/TemplateExport.yml"
+    Template:
+      $ref: "./openapi/src/common/schemas/Template.yml"
```
Comment (codyshepherd) [inline api/contract/cli.yml:261]: Is it a problem that these `openapi/src/` paths aren't present in `api/contract`?

File: cmd/influx/export.go [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+	influx export --stack-id $STACK_ID --buckets $BUCKET_ID
+
+All of the resources are supported via the examples provided above. Provide the
+resource flag and then provide the IDs.
+
+For information about exporting InfluxDB templates, see
+https://docs.influxdata.com/influxdb/latest/reference/cli/influx/export/`,
+		Flags: append(
+			commonFlagsNoPrint(),
+			&cli.StringFlag{
```
Comment (codyshepherd) [inline cmd/influx/export.go:69]: minor thing, but these could be sorted in lex order to make them easier to find

**Review by codyshepherd — APPROVED** [review]

File: cmd/influx/export.go (hunk 2 above)
Comment (danxmoran) [reply cmd/influx/export.go:69]: Opened #133 to address this in general (it's a problem across commands).

File: api/contract/cli.yml (hunk 1 above)
Comment (danxmoran) [reply api/contract/cli.yml:261]: Hmmm where did you see the paths weren't present? `openapi` is a git submodule so it takes a `git submodule update --init` for the paths to appear locally. The UI renders the folder as a link to the other repo.

**Pull request merged**